scheduler/graphql_mid/schema.py

```diff
@@ -22,6 +26,8 @@
 # TODO: This variables need a Redis cache to work with different mutations correctly.
 sources = Sources()
 event_queue = EventQueue(frozenset([NightIndex(i) for i in range(3)]), ALL_SITES)
```

Comment:
Just a question here: should `event_queue` be set explicitly to three nights? It seems like this should be calculated, no?